nilearn/input_data/multi_nifti_masker.py

```diff
@@ -121,18 +130,18 @@ class MultiNiftiMasker(NiftiMasker, CacheMixin):
     """
 
     def __init__(self, mask_img=None, smoothing_fwhm=None,
-                 standardize=False, detrend=False,
-                 low_pass=None, high_pass=None, t_r=None,
-                 target_affine=None, target_shape=None,
-                 mask_strategy='background', mask_args=None, dtype=None,
-                 memory=Memory(cachedir=None), memory_level=0,
-                 n_jobs=1, verbose=0
-                 ):
+                 standardize=False, standardize_strategy='zscore',
```

Comment:
May be we need standardize_strategy=None when standardize=False ? What do you think ?

Reply:
So when you set standardize to True, you _always_ need to give a string for what standardize strategy you want? Or do you want to replace None with 'zscore' if standardize =True?

I would prefer to keep things like this.

Reply:
By default, it standardizes or not ?

Reply:
Isn't it simpler to have a single kw for `standardize` and `standardize_strategy`, where 'False' would not do anything and otherwise would indicate the chosen strategy.